guides/release/components/block-content.md

```diff
@@ -22,6 +22,24 @@ This is equivalent to explicitly naming the default block using the named block
 </ExampleComponent>
 ```
 
+Or consider the following modal component which utilizes a content block and a footer block.
+
+```handlebars
+<div class="modal">
+    <button type="button">Close modal</button>
+    <div class="modal__content">
+        <:content>
+            This block will be replaced with the `{{yield to="content"}}` `<Modal />` content.
+        </:content>
+    </div>
+    <footer class="modal__footer">
+        <:footer>
+            This block will be replaced with the `{{yield to="footer"}}` `<Modal />` content.
+        </:footer>
+    </footer>
+</div>
```

Comment:
```suggestion
<div class="modal" role="dialog">
    <button type="button">Close modal</button>
    <div class="modal__content">
        <:content>
            This block will be replaced with the `{{yield to="content"}}` `<Modal />` content.
        </:content>
    </div>
    <div class="modal__footer">
        <:footer>
            This block will be replaced with the `{{yield to="footer"}}` `<Modal />` content.
        </:footer>
    </div>
</div>
```

I looked at https://github.com/MelSumner/e-a11y-modal to check out some example accessible markup. MDN suggests that `footer` only is relevant if it's inside of a `section`, so that's why I suggest using a div here. I don't usually build modals from scratch myself, so please let me know if you are aware of other patterns for footer.